SimplenoteUITests/NoteList.swift

```diff
@@ -32,11 +32,11 @@ class NoteList {
     }
 
     class func getNoteCellHeight(_ noteName: String) -> CGFloat {
-        return NoteList.getNoteCell(noteName).frame.height
+        return NoteList.getNoteCell(noteName).frame.height.rounded()
```

Comment:
This was returning `80.99999999999997' consistently.  But we are looking for `81`.  close enough...